Tests/U029-New-TableOnStorageAccount.ps1

```diff
@@ -0,0 +1,70 @@
+Push-Location -Path $PSScriptRoot\..\PSScripts\
+
+Describe "New-TableOnStorageAccount unit tests" -Tag "Unit" {
+        
+    # Re-define the three Az cmdlets under test, as we can't mock them directly.
+    # They fire a ParameterBindingValidationException on both powershell core and powershell 5.
+    # suspect it's due to https://github.com/pester/Pester/issues/619
+    function New-AzStorageContext { 
+        [CmdletBinding()]
+        param($StorageAccountName, $StorageAccountKey)
+    }
+
+    function Get-AzStorageTable {
+        [CmdletBinding()]
+        param($Context, $Name)
+    }
+
+    function New-AzStorageTable {
+        [CmdletBinding()]
+        param($Context, $Name)
+    }
+
+    Mock New-AzStorageContext
+    Mock New-AzStorageTable
+    Mock Get-AzStorageTable -MockWith { return $null }
+
+    Context "When the table exists" {
+    
+        Mock Get-AzStorageTable -MockWith { 
+            return @{}
```

Comment:
Should this not include a property (looks at first glance like it's returning the same as the other mock though I understand that it's returning an empty hashtable rather than null)